app/questionnaire/router.py

```diff
@@ -179,57 +200,64 @@ def is_survey_complete(self):
 
         return True
 
-    def is_path_complete(self, path):
-        location = self._get_first_incomplete_location(path)
+    def is_path_complete(self, routing_path_block_ids):
+        location = self._get_first_incomplete_location(routing_path_block_ids)
```

Comment:
This method is passed a `RoutingPath` object and not the `routing_path_block_ids`